bedrock/firefox/templates/firefox/firstrun/new-firstrun.html

```diff
@@ -0,0 +1,40 @@
+{% extends "bare.html" %}
+
+{% block page_title %}{{ _('Welcome to Firefox') }}{% endblock %}
+
+{% block site_css %}
+  {% stylesheet 'firefox_new_firstrun' %}
+{% endblock %}
+
+{% block site_js %}
```

Comment:
If you change this to `{% block js %}` then you don't need to include `{% javascript 'common' %}` again here on the line below, as you'll no longer be overwriting it.